deploy-board/deploy_board/webapp/deploy_views.py

```diff
@@ -28,6 +28,36 @@
 DEFAULT_PAGE_SIZE = 30
 DEFAULT_ONGOING_DEPLOY_SIZE = 10
 
+def _get_running_deploys_count(request):
+    deploy_states = ["RUNNING"]  # current running deploys
+    page_size = 1                # only need to return 1 detail, the return structure has the "total" value
+    deployResult = deploys_helper.get_all(request, deployState=deploy_states, pageSize=page_size)
+    return deployResult['total']
+
+def _get_sidecars(request):
+    # returns a list of env id for sidecars which are identified by having a system priority
+    envs = environs_helper.get_all_sidecar_envs(request)
+    env_ids = []
+    for env in envs:
```

Comment:
for easier readability: list comprehension candidate

env_ids = [env['id'] for env in envs]